Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -79,6 +79,10 @@ internal class CustomerIOImplementation: CustomerIOInstance {
         identifier: String,
         body: RequestBody
     ) {
+        if identifier.isBlankOrEmpty() {
```

Comment:
```suggestion
        guard !identifier.isBlankOrEmpty() else {q
```

I prefer `guard` statements because the compiler forces you to include a `return` statement in the body. A great reminder to exit the function early.

Reply:
There is not logical difference between the current implementation and the suggestion, hence keeping the code as is. Thanks for the suggestion.